torch_geometric/explain/algorithm/utils.py

```diff
@@ -0,0 +1,58 @@
+from typing import Dict
```

Comment:
Is this a copy-paste? Wondering why it doesn't pick up the diff :(

Reply:
yes it's just moving utilities from the old `Explainer` class